app/src/chat_engine.py

```diff
@@ -50,12 +50,15 @@ def create_engine(engine_id: str) -> ChatEngineInterface | None:
 
 # Subclasses of ChatEngineInterface can be extracted into a separate file if it gets too large
 class GuruBaseEngine(ChatEngineInterface):
+    datasets: list[str] = []
```

Comment:
should default dataset for this be `datasets: list[str] =["guru-snap"]`

Reply:
No, `GuruBaseEngine` is never instantiated. When the default engine (`GuruSnapEngine`) is created, it will have the correct `datasets` value.